src/libs/actions/Report.js

```diff
@@ -546,12 +546,13 @@ function updateReportWithNewAction(
     reportAction,
     notificationPreference = CONST.REPORT.NOTIFICATION_PREFERENCE.ALWAYS,
 ) {
+    const isAttachment = ReportUtils.isReportMessageAttachment(lodashGet(reportAction, ['message', 0], {}));
     const messageHtml = reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED
         ? lodashGet(reportAction, 'originalMessage.html', '')
         : lodashGet(reportAction, ['message', 0, 'html'], '');
 
     const parser = new ExpensiMark();
-    const messageText = parser.htmlToText(messageHtml);
+    const messageText = isAttachment ? `[${Localize.translateLocal('common.attachment')}]` : parser.htmlToText(messageHtml);
```

Comment:
@roryabraham I didn't have time to review this or look into why we were making this change, but this entire method is getting moved to the server and will be removed in my PR [here](https://github.com/Expensify/App/pull/9350/files).